src/main/java/frc/robot/RobotContainer.java

```diff
@@ -55,7 +55,7 @@ public RobotContainer() {
    */
   private void configureButtonBindings() {
     /* Driver Buttons */
-    zeroGyro.whenPressed(new InstantCommand(() -> swerveSubsystem.zeroGyro()));
+    zeroGyro.onTrue(new InstantCommand(() -> swerveSubsystem.zeroGyro()));
```

Comment:
Can you leave a comment on what onTrue does instead of whenPressed? That might help some of us.